Makefile

```diff
@@ -181,7 +181,6 @@ update-libcalico:
             fi;\
           OUTPUT=`mktemp`;\
           glide up --strip-vendor; glide up --strip-vendor 2>&1 | grep -v "golang.org/x/sys" | tee $$OUTPUT; \
```

Comment:
I think if we're going to get rid of this we should also remove the other logic re: $$OUTPUT, since it's not used anymore.